tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -471,7 +471,7 @@ class PerfIssuePlatformIssueUniqueUserFrequencyConditionTestCase(
 
 
 @freeze_time((now() - timedelta(days=2)).replace(hour=12, minute=40, second=0, microsecond=0))
-@region_silo_test
+@region_silo_test(stable=False)
```

Comment:
This should be safe to mark `stable=True`, it passed in the most recent test run: https://github.com/getsentry/sentry/actions/runs/6933129416